examples/contrib/notebooks/FastaiLRFinder_MNIST.ipynb

```diff
@@ -0,0 +1,677 @@
+   "outputs": [],
+   "source": [
+    "mnist_pwd = \"data\"\n",
+    "batch_size= 256"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "transform = transforms.Compose([transforms.ToTensor(), transforms.Normalize((0.1307,), (0.3081,))])\n",
+    "\n",
+    "trainset = MNIST(mnist_pwd, train=True, download=True, transform=transform)\n",
+    "trainloader = DataLoader(trainset, batch_size=batch_size, shuffle=True, num_workers=4)\n",
+    "\n",
+    "testset = MNIST(mnist_pwd, train=False, download=True, transform=transform)\n",
+    "testloader = DataLoader(testset, batch_size=batch_size * 2, shuffle=False, num_workers=0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "class Net(nn.Module):\n",
+    "    def __init__(self):\n",
+    "        super(Net, self).__init__()\n",
+    "        self.conv1 = nn.Conv2d(1, 10, kernel_size=5)\n",
+    "        self.conv2 = nn.Conv2d(10, 20, kernel_size=5)\n",
+    "        self.conv2_drop = nn.Dropout2d()\n",
+    "        self.fc1 = nn.Linear(320, 50)\n",
+    "        self.fc2 = nn.Linear(50, 10)\n",
+    "\n",
+    "    def forward(self, x):\n",
+    "        x = F.relu(F.max_pool2d(self.conv1(x), 2))\n",
+    "        x = F.relu(F.max_pool2d(self.conv2_drop(self.conv2(x)), 2))\n",
+    "        x = x.view(-1, 320)\n",
+    "        x = F.relu(self.fc1(x))\n",
+    "        x = F.dropout(x, training=self.training)\n",
+    "        x = self.fc2(x)\n",
+    "        return F.log_softmax(x, dim=1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ignite.engine import create_supervised_trainer, create_supervised_evaluator\n",
+    "from ignite.metrics import Loss, Accuracy\n",
+    "from ignite.contrib.handlers import FastaiLRFinder, ProgressBar"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Training loss (fastai)\n",
+    "\n",
+    "This learning rate test range follows the same procedure used by fastai. The model is trained for `num_iter` iterations while the learning rate is increased from its initial value specified by the optimizer algorithm to `end_lr`. The increase can be linear (`step_mode=\"linear\"`) or exponential (`step_mode=\"exp\"`); linear provides good results for small ranges while exponential is recommended for larger ranges."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "criterion = nn.NLLLoss()\n",
+    "model = Net()\n",
+    "optimizer = optim.SGD(model.parameters(), lr=3e-4, momentum=0.9)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
```

Comment:
Can we remove the state of this notebook from git?

Reply:
What do you mean by the state ? Outputs ?

Reply:
Yes, outputs and execution count

Reply:
I think it is intersting to see the results of the execution, even if jupyter widgets are badly rendered on github...

Reply:
The point why I ask to remove them is not that they are badly rendered (although this is true as well), but that results increase the repository size. And since they can basically be or arbitrary size, I would prefer to have no results at all in this repo and maybe link to colab instead... But if you want to keep them, that's fine

Reply:
That's a good point! Let's keep them for now, but remove everywhere once colab are ready.